.github/workflows/tnf-image.yaml

```diff
@@ -0,0 +1,128 @@
+name: 'Publish the `test-network-function` image (latest release only)'
+on:
+  # Run the workflow when a new release gets published
+  release:
+    types: [published]
+  # Run the workflow every day at 5 am UTC (1 am EST, 7am CET)
+  # This is useful for keeping the image up-to-date with security
+  # patches provided in the UBI.
+  # Disclaimer: There is no guarantee that scheduled workflows will
+  # run at the predefined time, if at all. The delay is usually
+  # around 10-30 minutes.
+  schedule:
+    - cron: '0 5 * * *'
```

Comment:
This will result in the TNF container image being rebuilt and pushed to Quay.io once a day.
That means a new entry in the [Tag history](https://quay.io/repository/testnetworkfunction/test-network-function?tab=history) will be added every day.

If we want to avoid that, there is a REST API that Quay.io exposes we could use to fetch the information about results of a security scan for a given image. This could be queried in a separate workflow, that would then trigger `tnf-image.yaml` only when a rebuild is necessary.

I suggest we stick with the current approach (or we could remove the `schedule` trigger completely), and treat the suggested improvements to the _vulnerability discovery and purging_ process as a separate story for future investigation.

Reply:
Sounds logical to me.